example/storybook/stories/BrandConfigProvider/BrandConfigProvider.stories.tsx

```diff
@@ -53,4 +56,48 @@ storiesOf('BrandConfigProvider', module)
         <ExampleBox message="Text inside a box" styles={componentStyles} />
       </BrandConfigProvider>
     );
+  })
+
+  .add('Icons', () => {
+    function IconExample({ name }: { name: string }) {
+      const { [name]: Icon } = useIcons();
+
+      return <Icon />;
+    }
+
+    return (
+      <BrandConfigProvider
```

Comment:
I see you've deleted all the svgs from the icons folder so I'm wondering if we still have default icons defined for the example app itself.